modules/serverless-rn-new-features.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// serverless/release-notes.adoc
+
+[id="serverless-rn-new-features_{context}"]
+
+////
+**Feature** – Describe the new functionality available to the customer.  For enhancements, try to describe as specifically as possible where the customer will see changes.
+**Reason** – If known, include why has the enhancement been implemented (use case, performance, technology, etc.).   For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.  There may not have been a 'problem' previously, but system behaviour may have changed.
+**Result** – If changed, describe the current user experience
+////
+
+== New features Technology Preview 1
```

Comment:
s/New features Technology Preview 1/New features _in_ Technology Preview 1/

And is it 1 or version 1? I don't know how we version our Tech Previews.